### DISCUSSION TITLE
Pytket 1.11.1 is missing for Python 3.8

### DISCUSSION THREAD

**Discussion by hamidelmaazouz** [CQCL/tket#786] [General]
Hello @cqc-alec,

I can't find any wheel for PyTKet 1.11.1 for Python 3.8 at https://pypi.org/project/pytket/
Can you point me to where I can find them ?
If that's intended, can you build and release the required packages for python 3.8 ?

Best,

**Comment by cqc-alec** [comment]
Hi, we no longer support 3.8 -- just 3.9, 3.10 and 3.11. (Sounds like you managed to build the wheel yourself though?)

**Reply by hamidelmaazouz** [reply]
> Sounds like you managed to build the wheel yourself though?

Yes, I was getting the issue described at https://github.com/CQCL/tket/issues/788 but as it turned out it's not Python related.

Best,
